libraries/core_libs/consensus/include/dag/dag.hpp

```diff
@@ -24,8 +24,8 @@
 #include "common/types.hpp"
```

Comment:
can we split this file to dag and dag_mngr? I do not really like combine those two things into one cpp/hpp

Reply:
Done